src/main/java/thePackmaster/cards/intriguepack/Spotlight.java

```diff
@@ -0,0 +1,23 @@
+package thePackmaster.cards.intriguepack;
+
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.actions.intriguepack.SpotlightAction;
+import thePackmaster.util.Wiz;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class Spotlight extends AbstractIntrigueCard {
+    public final static String ID = makeID("Spotlight");
+
+    public Spotlight() {
+        super(ID, 0, CardType.SKILL, CardRarity.UNCOMMON, CardTarget.NONE);
```

Comment:
Seems extremely powerful at 0 cost and non exhaust when compared to Hologram or Seek. Might want to increase the cost to 1